src/3box.js

```diff
@@ -173,7 +174,10 @@ class ThreeBox {
     if (serializedMuDID) {
       muportDID = new MuPort(serializedMuDID)
     } else {
-      const entropy = (await utils.openBoxConsent(address, web3provider)).slice(2, 34)
+      const sig = await utils.openBoxConsent(address, web3provider)
+      const entropy = sha256(sig.slice(2))
```

Comment:
The sha356Multihash is not used here because the entropy string becomes to long right?

Reply:
hmm actually didn't check if bip39.entropyToMnemonic require 32, it would for 12 word I guess. But mainly don't use it since it doesn't contribute any useful material as input to entropyToMnemonic, and doesn't seem to provide any useful intermediate step, unless if could in the future

Reply:
Yeah, that makes sense :)